sql/core/src/main/scala/org/apache/spark/sql/streaming/StreamingQueryException.scala

```diff
@@ -36,8 +36,8 @@ class StreamingQueryException private[sql](
     @transient val query: StreamingQuery,
     val message: String,
     val cause: Throwable,
-    val startOffset: Option[Offset] = None,
-    val endOffset: Option[Offset] = None)
+    val startOffset: Option[OffsetSeq] = None,
```

Comment:
Actually, we made a mistake in 2.0. We dont expose Offset in public API but this exposes Offset. This is wrong

Idea: is we replace startOffset with `startOffsetsDesc: String` (similar to `offsetDesc` in `SourceStatus`)